dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageChecker.java

```diff
@@ -219,15 +224,21 @@ private JarLinkageReport findInvalidReferences(Path jarPath, SymbolTable jarSymb
   }
 
   /**
-   * Scans class files in the jar file and creates {@link SymbolTable} populated with class names
+   * Scans class files in the jar file and returns a {@link SymbolTable} populated with class names
    * defined within the jar and symbolic references to other classes outside the jar.
    *
-   * @param jarFilePath absolute path to a jar file.
-   * @return Table containing symbol references and classes defined in the jar file.
+   * @param jarFilePath absolute path to a jar file
+   * @return symbol references and classes defined in the jar file
```

Comment:
This `@return` duplicates information already in the summary, so you can leave it out if you want.

And I see it hasn't been updated along with the summary. That's even more reason to remove it. It's hard to remember to update information everywhere it's repeated.

Reply:
Removed `@return`.